app/subscriber/src/store/hooks/subscriber/interfaces/IMinisterModel.ts

```diff
@@ -1,7 +1,6 @@
 import { ISortableModel } from 'tno-core';
 
 export interface IMinisterModel extends ISortableModel<number> {
-  name: string;
```

Comment:
Thanks for cleaning up.  @jdtoombs note that inherited properties don't need to be duplicated.